### ISSUE TITLE
Dynamic import unable to load requested module

### ISSUE THREAD

**Issue by mikart143** [microsoft/ClearScript#339]
Hello!
I am experimenting with ClearScript and I am unable to resolve one issue.

I was trying to invoke metod from module but, as the description of Invoke function says: only globally available functions and procedures.

I created special bootstraping script which does not seem to work properley.

I have such Program.cs:
```
public static class Program
    {
        static async Task Main(string[] args)
        {
            using (var engine = new V8ScriptEngine(V8ScriptEngineFlags.EnableDynamicModuleImports))
            {
                engine.DocumentSettings.AccessFlags = DocumentAccessFlags.EnableAllLoading;

                engine.ExecuteDocument("bootstrap.js", ModuleCategory.Standard);

                var b = await engine.Invoke("bootstrapModule", "./test.js", "testScript", new object[] { }).ToTask();

                Console.WriteLine(b as string);
            }
        }

    }
```

And such bootstrap.js
```
export async function bootstrapModule(path, funcName, params) {

    let importedModule;
    try {
        importedModule = await import(path);
    } catch (ex) {
        return 'err: ' + ex;
    }

    return importedModule[funcName].apply(null, params);
}

globalThis.bootstrapModule = bootstrapModule;
```

and also test scripts
```
//test.js
import { testScript2 } from './test2.js';

export function testScript() {
    return testScript2();
}

//test2.js
export function testScript2() {
    return 'addddsdf';
}
```

All the time I am receiving only error: Error: Invalid module load request. I am not sure why this occure. I think that this has problem with function from bootstrap.js that is "exported" globally via globalThis.bootstrapModule.

**Comment by ClearScriptLib** [comment]
Hi @mikart143,

This is a bug in ClearScript. A script function invoked directly by the host – e.g., via `ScriptEngine.Invoke` – currently can't perform dynamic module import. Thanks for reporting it!

We'll fix that in the next release. In the meantime, you can work around the issue by avoiding `ScriptEngine.Invoke`. For example:

``` C#
var b = await engine.Evaluate("bootstrapModule('./test.js', 'testScript', [])").ToTask();
```

Good luck!

**Comment by mikart143** [comment]
@ClearScriptLib
Thank you! Works as expected 😄

**Comment by ClearScriptLib** [comment]
Fixed in [Version 7.2.2](https://github.com/microsoft/ClearScript/releases/tag/7.2.2).
